src/BenchmarkDotNet/Environments/BenchmarkEnvironmentInfo.cs

```diff
@@ -30,15 +30,15 @@ public class BenchmarkEnvironmentInfo
         protected BenchmarkEnvironmentInfo()
         {
             Architecture = RuntimeInformation.GetArchitecture();
-            RuntimeVersion = RuntimeInformation.GetRuntimeVersion();
+            RuntimeVersion = "wasm"; // RuntimeInformation.GetRuntimeVersion();
```

Comment:
Need to extend RUntimeInformation.GetRuntimeVersion so that it can detect if we are running wasm.